AltSwiftUI/AltSwiftUI/Source/Views/Collections/ViewCollectionIteration.swift

```diff
@@ -138,10 +138,12 @@ extension Array where Element == View {
             // Optional insert / update
             let oldOptionalView = oldView as? OptionalView
             let maxCount = Swift.max(optionalViewContent.count, oldOptionalView?.content?.count ?? 0)
-            for i in 0..<maxCount {
-                let subView = optionalViewContent[safe: i]
-                let oldSubView = oldOptionalView?.content?[safe: i]
-                iterateFullSubviewDiff(subView: subView, oldView: oldSubView, iteration: iteration, displayIndex: &displayIndex)
+            if maxCount > 0 {
```

Comment:
Do you need this if? I mean, `for i in 0..<0 {` will not execute anyway right? 🤔

Reply:
you are right, `0..<0` won't crash. Fixed.